save-common/build.gradle.kts

```diff
@@ -27,5 +27,10 @@ kotlin {
                 implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:${Versions.serialization}")
             }
         }
+        val jvmMain by getting {
+            dependencies {
+                implementation("org.springframework.boot:spring-boot-starter-data-jpa:${Versions.springBoot}")
```

Comment:
Could you check if there is smaller dependency on JPA API only? For example, the transitive dependency of `spring-data-jpa`?